spec/approvals_spec.rb

```diff
@@ -105,6 +105,11 @@ def filter(data)
     Approvals.verify json, :format => :json, :namer => namer
   end
 
+  it "verifies json where the pretty generation is different but the content is the same" do
```

Comment:
I would also add some tests how diffs show when you changed something, and the pretty generation is different of the approved and received file.

Reply:
as far as i saw the diff is generated by rspec by just receiving what is written in `approved.json` and `received.json` Not sure how i could test that.